web/src/lib/components/content/Img.svelte

```diff
@@ -0,0 +1,30 @@
+<script lang="ts">
+	import { imageOptimazerUrl } from '$lib/Constants';
+	import { imageOptimization } from '$lib/stores/Preference';
+	import { formatStyleFromObject } from '$lib/styles/FormatStyleFromObject';
+	import { Img, type ImgSrc } from 'svelte-remote-image';
+	export let src: string;
+	export let pathname: string;
```

Comment:
URL として渡した方が異なる値が入ったりせず安全そうです。

```suggestion
	export let url: URL;
```

Reply:
確かにそのとおりでした。修正しました。

https://github.com/SnowCait/nostter/pull/1287/commits/eb98f683c002a9665d1a7f0c5203e40777c1a3b5